docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -0,0 +1,871 @@
+the `output.logstash` section by uncommenting it: 
+
+[source,yaml]
+----
+#-------------------------- Elasticsearch output ------------------------------
+#output.elasticsearch:
+  # Array of hosts to connect to.
+  #hosts: ["localhost:9200"]
+.
+.
+.
+#----------------------------- Logstash output --------------------------------
+output.logstash:
+  # The Logstash hosts
+  hosts: ["localhost:5044"]
+----
+
+
+Save the file, then restart {metricbeat} to apply the changes.
+
+
+[float]
+[[logstash-setup]]
+==== Configure {ls} to read from the {beats} input
+
+{ls} provides {logstash-ref}/input-plugins.html[input plugins] for reading from
+a variety of inputs. In this tutorial, you create a {ls} pipeline configuration
+that reads from the {beats} input and sends events to the {es} output.
+
+To configure {ls}:
+
+. Create a new {ls} pipeline configuration file called `logstash.conf` that
+contains:
++
+--
+* An input stage that configures {ls} to listen on port 5044 for incoming {beats}
+connections.
+* An output stage that indexes events into {es}. The output stage also
+configures {ls} to write to the {metricbeat} index.
+--
++
+For example:
++
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+# The filter part of this file is commented out to indicate that it
+# is optional.
+# filter {
+#
+# }
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}" <1>
+  }
+}
+----
++
+This configuration allows you to route events to {ls}, where you have full
+access the {ls} capabilities for collecting, enriching, and transforming
+data.
+
+
+[float]
+==== Start {ls}
+
+Use the command that works with your system. If you installed {ls} as a deb or
+rpm package, make sure the config file is in the `config` directory.
+
+*deb:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+sudo /etc/init.d/logstash start
+----------------------------------------------------------------------
+
+*rpm:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+sudo service logstash start
+----------------------------------------------------------------------
+
+*mac:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+./bin/logstash -f logstash.conf
+----------------------------------------------------------------------
+
+*win:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+bin\logstash.bat -f logstash.conf
+----------------------------------------------------------------------
+
+NOTE: The default configuration for {beats} and {ls} uses plain TCP. For
+encryption you must explicitly enable SSL when you configure {beats} and {ls}.
```

Comment:
Is there somewhere they can find more information about that?  Maybe https://www.elastic.co/guide/en/elastic-stack-overview/master/encrypting-communications.html ?

Reply:
I'm good with removing the note if there's a security step.

Reply:
"plain TCP" is an implementation detail that I don't think is helpful here; how about:

~~~
NOTE: the default configuration for {beats} and {ls} does not use encryption and does not
provide secure communication; before sending sensitive data across the network, make sure
you enable {stack-ref}/encrypting-communications.html[encrypted communications]
between {beats} and {ls}.
~~~

Reply:
I'm removing the note from this section, but working some of Ry's language into the note that appears earlier because it's important to call out that communications are not secure until you explicitly turn on security.